### ISSUE TITLE
Example Not Working Properly cant event able to even login

### ISSUE THREAD

**Issue by ios3** [mike4aday/SwiftlySalesforce#89]
![screen shot 2018-08-28 at 7 59 23 pm](https://user-images.githubusercontent.com/39147630/44729681-f5f8bd80-aafc-11e8-873f-32eb1a781980.png)

**Comment by mike4aday** [comment]
@ios3 I need to update the consumer key value; will do that ASAP. In the meantime, you could replace the value for `consumerKey` on line 18 in `AppDelegate.swift` as follows:
```swift
let consumerKey = "3MVG91ftikjGaMd_SSivaqQgkiuG.Y1epLf6raoqyNRl6rh89ffM3ugG92nuAkbjDslWsiu6iVIfP2dgSJjIt"
```

**Comment by mike4aday** [comment]
@ios3 should be fixed now in version 7.0.3. Thanks.
